src/mame/drivers/pofo.cpp

```diff
@@ -56,7 +56,7 @@ namespace {
 
 #define LOG 0
 
-#define M80C88A_TAG     "u1"
+#define M80C88A_TAG     "maincpu"
```

Comment:
What’s the point of this?  It seems like gratuitous churn.

Reply:
It's in the commit message: The debugger stub needs this to work.

Reply:
Well that’s a problem with the debugger stub.  Nothing should be relying on fixed tags.